src/regions/ca/i18n/steps.fr.ts

```diff
@@ -109,16 +109,9 @@ export default {
   continue: 'Continuer',
   faq: {
     utterAskHasAdditionalQuestion: `
-    Nous sollicitons des questions relatives à la COVID-19.
-    Ces questions seront utilisées pour nous aider à raffiner et à compléter l'information sur la pandémie.
-    Écrivez une question que vous avez.
-
-    Votre aide est nécessaire et grandement appréciée.
-    `,
-    utterAskForQuestion: `Svp, écrivez une question`,
-    utterQuestionAsked: `
-    Avez-vous une autre question ?
+    Avez-vous une question relative à la COVID-19?
```

Comment:
`relative à` doesn't really work in French IMO, I'd consider `à propos de`, `concernant`, `vis-à-vis`, `entourant`, `associée à`, etc